publish.html

```diff
@@ -629,6 +629,57 @@
 <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
+<h2><a class="anchor" id="repository"></a>Using OMERO as a Public Repository</h2>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<p>OMERO can be used as a public data repository with either a custom web app or using the OMERO.gallery app.</p>
```

Comment:
"In addition to adding the Public User to allow access to your data, the appearance of OMERO.web can be customised or you can use a separate custom app to tailor the front end of your OMERO installation to your target audience."